inspirehep/base/static/js/search/search_vis.js

```diff
@@ -0,0 +1,246 @@
+ define(["jquery",
```

Comment:
Yes, it is still in progress. After I finished I will run jsbeautifier and it will be ok ;)